src/Pages/Home/Home.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from 'styled-components';
+import {Header, Container, Link, H1, Paragraph, BigButton, Login} from './styled';
+
+
+
+export const Home = function () {
+    return (
+        <Container>
+            <Header>

```

Comment:
Move `Header ` staff to separated component `HomePageHeader` (in HomePage folder)